app/javascript/controllers/navbar_controller.js

```diff
@@ -1,16 +1,17 @@
 import { Controller } from "@hotwired/stimulus"
 
-export default class extends Controller {
+const navbar = document.querySelector('nav'),
+      navMenu = navbar.querySelector('.nav-menu'),
+      navMenuBackground = navbar.querySelector('.nav-menu-background')
 
+export default class extends Controller {
     expand() {
-        const navbar = this.element,
-            nav_mobile_expand = navbar.querySelector('.nav-mobile-expand'),
-            icon_class_list = nav_mobile_expand.querySelector('i').classList
+        navMenu.style.display = 'flex'
+        navMenuBackground.style.display = 'block'
+    }
 
-        if (navbar.classList.toggle('expanded')) {
-            icon_class_list.replace('fa-bars', 'fa-times')
-        } else {
-            icon_class_list.replace('fa-times', 'fa-bars')
-        }
+    collapse() {
+        navMenu.style.display = 'none'
+        navMenuBackground.style.display = 'none'
     }
```

Comment:
How COOL are stimulus controllers? Is it just me or does this seem clearer than other javascript?